### ISSUE TITLE
How does this deal with contacts in Trash?

### ISSUE THREAD

**Issue by petednz** [systopia/de.systopia.selfservice#14]

**Comment by bjendres** [comment]
Hi @petednz. 

When sending out links, 'trashed' contacts are ignored (see [here](https://github.com/systopia/de.systopia.selfservice/blob/1.0-beta1/api/v3/Selfservice/Sendlink.php#L48-L60)), as well as in the ``getContactIDsWithSharedEmails`` function (see [here](https://github.com/systopia/de.systopia.selfservice/blob/1.0-beta1/CRM/Selfservice/HashLinks.php#L167)), which is used in the smarty token resolution logic.

Does that answer your question?

**Comment by petednz** [comment]
Yes thank you - i guess i was wondering if there was a way for a 'contact the office' type message to go out for such contacts - but am sure we can offer that as a PR if it becomes important to us
